core/tokens.go

```diff
@@ -0,0 +1,91 @@
+package core
+
+import (
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/mr-tron/base58/base58"
+	"github.com/segmentio/ksuid"
+	"github.com/textileio/textile-go/crypto"
+	"github.com/textileio/textile-go/repo"
+	"golang.org/x/crypto/bcrypt"
+)
+
+// CreateCafeToken creates a random developer access token, returns a base58 encoded version,
+// and stores a bcrypt hashed version for later comparison
+func (t *Textile) CreateCafeToken() (string, error) {
+	key, err := crypto.GenerateAESKey()
+	if err != nil {
+		return "", err
+	}
+
+	id := ksuid.New().String()
+	created := time.Now()
+	rawToken := key[:32]
+
+	safeToken, err := bcrypt.GenerateFromPassword(rawToken, bcrypt.DefaultCost)
+	if err != nil {
+		return "", err
+	}
+
+	err = t.datastore.CafeDevTokens().Add(
+		&repo.CafeDevToken{
+			Id:      id,
+			Token:   safeToken,
+			Created: created,
+		})
+	if err != nil {
+		return "", err
+	}
+
+	return id + "+" + base58.FastBase58Encoding(rawToken), nil
+}
+
+// CafeDevTokens lists all stored (bcrypt encrypted) dev tokens
+func (t *Textile) CafeDevTokens() ([]string, error) {
+	tokens := t.datastore.CafeDevTokens().List()
+	strings := make([]string, len(tokens))
+	for i, token := range tokens {
+		strings[i] = token.Id + "+" + base58.FastBase58Encoding(token.Token)
```

Comment:
This is a bit silly maybe? Probably don't even need this method to be honest. Could be replaced with a `Count` method instead?

Reply:
seems nice to have to me

Reply:
ok, let's keep it